src/questions/aave-other-features.yaml

```diff
@@ -251,5 +251,153 @@
       key: C
     - content: It is an alias for "bridging protocol"
       key: D
+- uuid: c09b82e3-fe01-4d4c-b377-f2ac02a463f5
+  type: MultipleChoice
+  content: Which of these changes would be made in Aave after GHO’s integration with Aave protocol?
+  hint: noHint
+  explanation: aTokens and debt tokens will be minted for GHO, Aave market on Ethereum would become the first facilitator, GHO will be borrowed at a stable rate determined by the DAO.
+  answerKeys:
+    - A
+    - D
+  subTopics:
+    - aave-gho
+  difficultyLevel: "Medium"
+  choices:
+    - content: aTokens and debt tokens will be minted for GHO
+      key: A
+    - content: New aTokens and debt tokens will not be minted for GHO
+      key: B
+    - content: The Aave protocol will become slower due to integration of a new stablecoin
+      key: C
+    - content: Aave market on Ethereum would become the first facilitator
+      key: D
+
+- uuid: c32c634a-47fe-4dbc-a485-e9e864ac702b
+  type: MultipleChoice
+  content: Which of these statements about GHO’s interest rate is accurate?
+  hint: noHint
+  explanation:  Interest rate will be set by the AaveDAO initially, It will be stable rate as opposed to variable rate, can be adapted to market conditions. The rates can be altered based on the community and governance 
+  answerKeys:
+    - A
+    - C
+    - D
+  subTopics:
+    - aave-gho
+  difficultyLevel: "Medium"
+  choices:
+    - content: It will be set by the AaveDAO initially
+      key: A
+    - content: The rates determined by AaveDAO will be final and cannot be changed
+      key: B
+    - content: It will be stable rate as opposed to variable rate, can be adapted to market conditions
```

Comment:
`It will be stable rate as opposed to variable rate, can be adapted to market conditions` improve the wording